web/src/Update.elm

```diff
@@ -133,6 +133,21 @@ update msg model =
             in
             { model | democracies = Dict.insert democId setDelegateState model.democracies } ! []
 
+        Debug str ->
+            let
+                currentLog =
+                    Dict.get lSKeys.debugLog model.localStorage ? ""
+
+                -- TODO: model.now does't update in real time.
```

Comment:
You can add a subscription to have model.now update. Don't have an example though. You can also set such a task to run every 1s or 10s or whatever.